z_systems_administration/zhmc/playbooks/inventory_example.yml

```diff
@@ -1,9 +1,13 @@
 ---
 # inventory.yml file for the sample playbooks of the IBM Z HMC Collection
 
-# The HMC inventory the sample playbooks will use:
-my_hmc_list:
+default:  # A group nickname
+
   hosts:
-    myhmc13:  # must match the key name in the vault file
+
+    myhmc13:  # An HMC nickname (must match the nickname in the vault file)
       ansible_host: 10.11.12.13
```

Comment:
does this ip address belong to IBM? if yes, maybe just set to something generic like _'your_hmc_ip_address'_